media-video/pipewire/pipewire-9999.ebuild

```diff
@@ -85,7 +92,7 @@ DEPEND="${RDEPEND}"
 # Ditto for DEPEND
 #	>=dev-util/vulkan-headers-1.1.69
 
-DOCS=( {README,INSTALL}.md NEWS )
+#DOCS=( {README,INSTALL}.md NEWS )
```

Comment:
Probably leftover?

Reply:
If you mean that installing {README,INSTALL}.md NEWS is a left-over from before good integration, I'm more than willing to agree and officially stop installing them. Just say so and they'll be gone. If you meant the commented out code - the dependencies are still valid, if we decided to turn back on the features they correspond to - they just make no sense to ship. In principle we could remove them completely.

Reply:
No, the other way. We need to install them.